test/testdata/dsl/rails/cattr_writer.rb

```diff
@@ -0,0 +1,25 @@
+# typed: strict
+
+class GoodUsages
+  extend T::Sig
+  cattr_writer :both, :foo
+  cattr_writer :no_instance, instance_accessor: false
+  cattr_writer :bar, :no_instance_writer, instance_writer: false
```

Comment:
Same thing here about `:bar` and my question about why it seems like some of these things are dupes of `cattr_accessor`

Reply:
jk about the question of duped tests ( https://github.com/sorbet/sorbet/pull/1252/files#r302289355 )